docs/OV_Runtime_UG/auto_device_selection.md

```diff
@@ -153,66 +149,66 @@ The following commands are accepted by the API:
 
    .. code-block:: python
 
-      ### New IE 2.0 API ###
-	  
+      ### OpenVINO Runtime API 2.0 ###
+
       from openvino.runtime import Core
       core = Core()
-      
+
       # Read a network in IR, PaddlePaddle, or ONNX format:
       model = core.read_model(model_path)
-      
+
       # Load a network to AUTO using the default list of device candidates.
       # The following lines are equivalent:
-      model = core.compile_model(model=model) 
+      compiled_model = core.compile_model(model=model)
       compiled_model = core.compile_model(model=model, device_name="AUTO")
       compiled_model = core.compile_model(model=model, device_name="AUTO", config={})
-      
+
       # You can also specify the devices to be used by AUTO in its selection process.
       # The following lines are equivalent:
-      compiled_model = core.compile_model(model=model, device_name="AUTO:CPU,GPU")
-      compiled_model = core.compile_model(model=model, device_name="AUTO", config={"MULTI_DEVICE_PRIORITIES": "CPU,GPU"})
-      
+      compiled_model = core.compile_model(model=model, device_name="AUTO:GPU,CPU")
+      compiled_model = core.compile_model(model=model, device_name="AUTO", config={"MULTI_DEVICE_PRIORITIES": "GPU,CPU"})
+
       # the AUTO plugin is pre-configured (globally) with the explicit option:
-      core.set_config(config={"MULTI_DEVICE_PRIORITIES":"CPU,GPU"}, device_name="AUTO")
-    
-.. tab:: Python legacy API
+      core.set_property(device_name="AUTO", properties={"MULTI_DEVICE_PRIORITIES":"GPU,CPU"})
+
+.. tab:: Python API
 
    .. code-block:: python
 
-      ### API before 2022.1 ###
+      ### Inference Engine API ###
       from openvino.inference_engine import IECore
       ie = IECore()
-      
+
       # Read a network in IR, PaddlePaddle, or ONNX format:
       net = ie.read_network(model=path_to_model)
-      
+
       # Load a network to AUTO using the default list of device candidates.
       # The following lines are equivalent:
       exec_net = ie.load_network(network=net)
       exec_net = ie.load_network(network=net, device_name="AUTO")
       exec_net = ie.load_network(network=net, device_name="AUTO", config={})
-      
+
       # You can also specify the devices to be used by AUTO in its selection process.
       # The following lines are equivalent:
-      exec_net = ie.load_network(network=net, device_name="AUTO:CPU,GPU")
-      exec_net = ie.load_network(network=net, device_name="AUTO", config={"MULTI_DEVICE_PRIORITIES": "CPU,GPU"})
-      
+      exec_net = ie.load_network(network=net, device_name="AUTO:GPU,CPU")
+      exec_net = ie.load_network(network=net, device_name="AUTO", config={"MULTI_DEVICE_PRIORITIES": "GPU,CPU"})
+
       # the AUTO plugin is pre-configured (globally) with the explicit option:
-      ie.SetConfig(config={"MULTI_DEVICE_PRIORITIES", "CPU,GPU"}, device_name="AUTO");
+      ie.set_config(config={"MULTI_DEVICE_PRIORITIES":"GPU,CPU"}, device_name="AUTO");
 
 @endsphinxdirective
```

Comment:
let's refer to openvino_docs_OV_UG_query_api.html#get-a-set-of-available-devices in the text below instead of writing manual snippets

Reply:
@ilya-lavrenov  I didn't get the link, could you shared it again

Reply:
I have put my proposition with the link in a comment below. In this case we can remove the code.